css-pseudo-4/Overview.bs

```diff
@@ -1634,6 +1634,32 @@ Styling form control pickers: the ''::picker()'' pseudo-element</h3>
 &lt;/style>
   </pre>
 
+<h3 id="select-arrow">
+Select dropdown icon: the ''::select-arrow'' pseudo-element</h3>
+
+  The ''::select-arrow'' pseudo-element only matches when the <a>originating
+  element</a> is a <{select}> element.
+
+  ''::select-arrow'' is a [=part-like pseudo-element=], so there is no
+  restriction on which properties apply to the ''::select-arrow'' pseudo-element.
+
+  ''::select-arrow'' generates a box as if it was an immediate child of its
```

Comment:
Are there conditions on when this generates a box?  Does it only do something for `appearance: base-select`?

Reply:
Sure, I added the restriction that there must be a select element with base appearance.